build/grunt.js

```diff
@@ -23,11 +85,13 @@ module.exports = function(grunt) {
    *                       using an object constructed from `bannerCommonData`
    *                       and the `data` argument.
    */
-  let createLicenseProcessor = (data) => function () {
-    return grunt.template.process(license, {
-      data: _.merge({}, bannerCommonData, data)
-    });
-  };
+  function createLicenseProcessor(data) {
```

Comment:
and this change is just stylistic, if I'm reading it correctly?
From let+()=>{} to function?